### ISSUE TITLE
Missing file on my system

### ISSUE THREAD

**Issue by quicklisp** [marcoheisig/Petalisp#13]
http://report.quicklisp.org/2020-07-31/failure-report/petalisp.html#petalisp has a log. I get this:

``` 
The file #P"/sys/devices/system/cpu/cpu0/cache/power/level" does not exist: No such file or directory
```

**Comment by quicklisp** [comment]
I'm using Debian 10 on an amd64 system.

**Comment by marcoheisig** [comment]
Should be addressed by 8f5a83c58eabce54a3c8d83f0c020f044cfca14b.  Thanks for reporting!
